tools/build_config/Jenkinsfile

```diff
@@ -1,18 +1,40 @@
 #!groovy
 
+def getRepoURL() {
```

Comment:
https://stackoverflow.com/questions/36547680/how-to-do-i-get-the-output-of-a-shell-command-executed-using-into-a-variable-fro suggests you can directly access the output of the shell command in Jenkins/Groovy without needing an intermediate file.:
```
return sh(script: 'git config --get remote.origin.url', returnStdout: true)
```
I you need the `trim()` you can probably append that.